dotnet-test.targets

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- This is a hack to make sure that the right file is being picked up to be run for tests using the xproj file name. The TestTargetFilename should be set correctly to the name inside project.json, but it is not. -->
+  <PropertyGroup>
+  	<XunitArguments>$(MSBuildProjectName).dll $(XunitOptions)</XunitArguments>
+
+    <TestProgram Condition="'$(TestHost)'!=''">$(TestHost)</TestProgram>
+    <TestArguments Condition="'$(TestHost)'!=''">$(XunitExecutable) $(XunitArguments)</TestArguments>
+
+    <TestProgram Condition="'$(TestHost)'==''">$(XunitExecutable)</TestProgram>
+    <TestArguments Condition="'$(TestHost)'==''">$(XunitArguments)</TestArguments>
+
+    <TestCommandLine>$(TestProgram) $(TestArguments) {XunitTraitOptions}</TestCommandLine>
+  </PropertyGroup>
+
+  <Target Name="CopyTestToTestDirectory" 
+          BeforeTargets="RunTestsForProject">
+    <Message Text="Running CopyTestToTestDirectory: $(TestPath)%(TestTargetFramework.Folder)" Importance="High" />
+
+    <Exec Command="$(DotnetExe) publish &quot;$(ProjectJson)&quot; -o &quot;$(TestPath)%(TestTargetFramework.Folder)&quot; -f dnxcore50 -r win7-x64" />
```

Comment:
Absolutely. I am eagerly waiting for that. How is the assembly load work coming along.